index.html

```diff
@@ -822,6 +830,22 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
       </div>
     </p>
     </div>
+    <h3>Example</h3>
+    <div>
+    <p>This example shows how to monitor external background blur changes.</p>
+    <pre class="example">
+const stream = await navigator.mediaDevices.getUserMedia({video: true});
+const [track] = stream.getVideoTracks();
+let {backgroundBlur} = track.getSettings();
+applyBlurInSoftwareInstead(!backgroundBlur);
+
+track.addEventListener("configurationchange", () => {
+  if (backgroundBlur != track.getSettings().backgroundBlur) {
+    backgroundBlur = track.getSettings().backgroundBlur;
+    applyBlurInSoftwareInstead(!backgroundBlur);
+  }
+});</pre>
+    </div>
```

Comment:
nit: It's weird to see the \</pre\> as less indented than the \</div\>. The following indentation seems preferrable imho:
```
    <h3>Example</h3>
    <div>
      <p>This example shows how to monitor external background blur changes.</p>
      <pre class="example">
       const stream = await navigator.mediaDevices.getUserMedia({video: true});
       const [track] = stream.getVideoTracks();
       let {backgroundBlur} = track.getSettings();
       applyBlurInSoftwareInstead(!backgroundBlur);

       track.addEventListener("configurationchange", () => {
         if (backgroundBlur != track.getSettings().backgroundBlur) {
           backgroundBlur = track.getSettings().backgroundBlur;
           applyBlurInSoftwareInstead(!backgroundBlur);
         }
       });
      </pre>
    </div>
  </section>
```

Reply:
Fixed in https://github.com/w3c/mediacapture-extensions/pull/89/commits/3e48bd26c27ad18b669bbb18c10b4fa26a267e95